cve_bin_tool/util.py

```diff
@@ -72,7 +76,7 @@ class VersionInfo(NamedTuple):
 
 
 class CVEData(defaultdict):
-    def __missing__(self, key):
+    def __missing__(self, key: str) -> Union[List[CVE], Set[str]]:
```

Comment:
We can simplify this due to the `annotations` imported above (see my other suggestion):
```suggestion
    def __missing__(self, key: str) -> list[CVE] | set[str]:
```

Reply:
Checked it again, works with `annotations` imported.